bin/cli.js

```diff
@@ -107,9 +107,7 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		 * used in place of process.stout
 		 */
 		const stdout = argv.silent
-			? {
-				write: () => {}
-			  } // eslint-disable-line
+			? { write: () => {} } // eslint-disable-line
```

Comment:
Do we have to keep the disable line here?

Reply:
Nope, and good catch! :)